README.md

```diff
@@ -277,3 +278,71 @@ See the [metrics documentation](metrics).
 
 ## Kata Admission controller webhook
 See the [webhook documentation](kata-webhook).
+
+## Using Vagrant to test your code changes
+
+It is strongly recommended that you test your changes locally before opening
+the pull request as this can save people's time and CI resources. Because
+testing Kata Containers involve complex build and setup instructions, scripts
+on the `.ci` directory are created to easy and provide a reproducible process; but they
```

Comment:
missing word before "easy"? to be easy to use?

Reply:
I meant "ease"